README.md

```diff
@@ -48,6 +48,43 @@ $ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/rel
 $ oc apply -f bundle/manifests
 ```
 
+### Create Project
+
+Create the project on jira-service-desk with the following custom resource:
+
+```yaml
+apiVersion: jiraservicedesk.stakater.com/v1alpha1
+kind: Project
+metadata:
+  name: <Name>
+  namespace: default
+spec:
+  name: <ProjectName>
+  key: <ProjectKey>
+  projectTypeKey: <ProjectTypeKey>
+  projectTemplateKey: <ProjectTemplateKey>
+  description: <ProjectDescription>
+  assigneeType: <ProjectAssigneeType>
+  leadAccountId: <ProjectLeadAccountId>
+  url: <ProjectUrl>
```

Comment:
Don't add manifest here. Just refer the manifest from `examples/project`
The issue with this is that we'll have to update it in multiple places whenever we update Project CRD